Tribler/Test/Core/TorrentChecker/test_torrentchecker.py

```diff
@@ -1,35 +1,32 @@
 import socket
 import time
+
 from twisted.internet.defer import Deferred, inlineCallbacks
 
 from Tribler.Core.CacheDB.SqliteCacheDBHandler import TorrentDBHandler
 from Tribler.Core.Category.Category import Category
-from Tribler.Core.Config.tribler_config import TriblerConfig
 from Tribler.Core.Modules.tracker_manager import TrackerManager
-from Tribler.Core.Session import Session
 from Tribler.Core.TorrentChecker.session import HttpTrackerSession, UdpSocketManager
 from Tribler.Core.TorrentChecker.torrent_checker import TorrentChecker
 from Tribler.Core.simpledefs import NTFY_TORRENTS
-from Tribler.Test.Core.base_test import TriblerCoreTest, MockObject
-from Tribler.Test.twisted_thread import deferred
+from Tribler.Test.Core.base_test import MockObject
+from Tribler.Test.test_as_server import TestAsServer
 from Tribler.community.popularity.repository import TYPE_TORRENT_HEALTH
-from Tribler.pyipv8.ipv8.util import blocking_call_on_reactor_thread
 
 
-class TestTorrentChecker(TriblerCoreTest):
+class TestTorrentChecker(TestAsServer):
```

Comment:
Why change the super class?

Reply:
Because `TestAsServer` provides correct initialization of Tribler `Session`. Before, the test was initiating the `Session` manually (see line 31 of the previous file). It is cleaner and more consistent to just use `TestBase` whenever we want a `Session` in our tests.